### ISSUE TITLE
Support multiple trust certificate paths for `X509CredentialConfig`

### ISSUE THREAD

**Issue by mmoayyed** [vt-middleware/ldaptive#193]
Given the following code snippet:

```java
var cfg = new X509CredentialConfig();
cfg.setTrustCertificates(getTrustCertificatesAsString());
cc.setSslConfig(new SslConfig(cfg));
```

It appears that, as of ldaptive 2.0.1, `X509CredentialConfig#setTrustCertificates()` generally can only support a single resource when working with a path as a String. This feels odd, specially since the method name `setTrustCertificates()` implies support for multiple certificates should be allowed. I suppose that is still valid if one were to put together a keystore of some kind as a single resource to contain multiple certificates.

For convenience, I want to suggest that `AbstractCredentialReader` may be modified to accept multiple individual paths, separated by comma/space/etc so that multiple trust certificates can be found and set on the credential configuration without the need to combine them in a keystore. 

Something like this:

```java
var cfg = new X509CredentialConfig();
cfg.setTrustCertificates(List.of("cert1.cer", "cert2.cer"));
cc.setSslConfig(new SslConfig(cfg));
```

This feels more intentional and somewhat more obvious, and does require less work from the ldaptive user (no one likes messing about with keystores!) I'd be happy to send a pull request for review, if this is a reasonable enhancement.

**Comment by dfish3r** [comment]
> It appears that, as of ldaptive 2.0.1, X509CredentialConfig#setTrustCertificates() generally can only support a single resource when working with a path as a String. This feels odd, specially since the method name setTrustCertificates() implies support for multiple certificates should be allowed.

In this case the resource can contain multiple certificates, the most common being multiple PEM encoded certificates. You're correct, it doesn't accept multiple paths, but a single file can contain multiple certs.

> For convenience, I want to suggest that AbstractCredentialReader may be modified to accept multiple individual paths, separated by comma/space/etc so that multiple trust certificates can be found and set on the credential configuration without the need to combine them in a keystore.

I'm not opposed if you've got a use case where trust anchors need to be in separate files. 

> This feels more intentional and somewhat more obvious, and does require less work from the ldaptive user (no one likes messing about with keystores!) I'd be happy to send a pull request for review, if this is a reasonable enhancement.

Pull requests are always welcome :)

**Comment by mmoayyed** [comment]
Thanks much for the review! 

I am not sure this qualifies as a legit use case; I did have a number of DER-encoded .crt/.cer files whose path I thought I could just individually be listed and passed onto `setTrustCertificates`. Of course, there is any number of things I could have done to combine them and put them into some sort of keystore, but it would have been inconvenient for the lazy me, specially more so, because I then had to teach someone else how to do that, and why that should be done. So this is mostly about saving lives rather than a real limitation per se.

I'll try to put something together in the next few days for you to review.
